### PR TITLE
chore: Renames/Updates breakpoints tokens

### PR DESCRIPTION
**Pull request by hellofanny**
## What's the purpose of this pull request?

Updates all the breakpoint tokens to follow the same suffix pattern: `-mobile` / `-tablet` / `-notebook` / `-desktop` 

## How does it work?

Some tokens were still using `-mob` and `-desk` suffix. Now should be updated.

## How to test it?

No visual changes were applied. Everything should work as before.

## Checklist

<em>You may erase this after checking them all :wink:</em>

**PR Title and Commit Messages**
- [x] PR title and commit messages follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) specification
  - Available prefixes: `feat`, `fix`, `chore`, `docs`, `style`, `refactor`, `perf`, and `test`

**PR Description**
- [x] Added a label according to the PR goal - `Breaking change`, `Features`, `Bug fixes`, `Chore`, `Documentation`, `Style changes`, `Refactoring`, `Performance`, and `Test`
- [x] Added the component, hook, or path name in-between backticks (\`\`) - *if applicable, e.g., `ComponentName` component, `useWindowDimensions` hook*

**Dependencies**
- [ ] Committed the `yarn.lock` and `bun.lockb` file when there were changes to the packages

**Documentation**
- [x] PR description
- [x] Added to/Updated the Storybook - *if applicable*
- [ ] For documentation changes, ping `@carolinamenezes` or `@PedroAntunesCosta` to review and update

### CODE REVIEW

**Review by renatamottam — APPROVED** [review]
LGTM!

**Review by filipewl — APPROVED** [review]
Very quick to [review](https://github.com/vtex-sites/nextjs.store/pull/209/files?w=1)! Looks good to me!

File: src/components/sections/Hero/hero.scss [hunk 1]
```diff
@@ -6,36 +6,36 @@
...
-  --fs-hero-primary-title-size         : var(--fs-text-size-title-huge);
+  --fs-hero-primary-img-height-mobile       : #{rem(240px)};
+  --fs-hero-primary-img-height-desktop   : #{rem(464px)};
+  --fs-hero-primary-title-size           : var(--fs-text-size-title-huge);
 
   // Secondary
-  --fs-hero-secondary-img-height-mob   : #{rem(180px)};
-  --fs-hero-secondary-img-height-desk  : #{rem(227px)};
-  --fs-hero-secondary-title-size       : var(--fs-text-size-title-page);
+  --fs-hero-secondary-img-height-mobile     : #{rem(180px)};
```
Comment (eduardoformiga) [inline src/components/sections/Hero/hero.scss:24]: Nitpicking: the colon isn't aligned.

**Review by eduardoformiga — APPROVED** [review]

File: src/components/sections/Hero/hero.scss (hunk 1 above)
Comment (hellofanny) [reply src/components/sections/Hero/hero.scss]: gotcha! Fixed [here](https://github.com/vtex-sites/nextjs.store/pull/209/commits/13d19aa8b8b003e81f7d013e962a8fb329b1d5a7)! Thanks 🚀

**Pull request merged**